cli/internal/run/run.go

```diff
@@ -997,9 +997,6 @@ func (ec *execContext) exec(ctx gocontext.Context, packageTask *nodes.PackageTas
 	// Setup tracer
 	tracer := ec.runState.Run(packageTask.TaskID)
 
-	// Create a logger
-	targetUI := &cli.BasicUi{}
-
```

Comment:
This was an incomplete migration from here:
https://github.com/vercel/turborepo/pull/2126/files#diff-a79bb9f30997e3382bc1428116c8d0637778720674f22d669b2d7862607e535aR996

`prefixedUI` has the same construction as `targetUI` did originally and is vestigial at this point.

Reply:
it was intentional, i just didn't know what I was doing I guess 😢 . I thought `BasicUi` had what it needed to write logs by itself.